gammapy/scripts/tests/test_make_bg_cube_models.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from astropy.tests.helper import pytest, remote_data
+from ...datasets import get_path
+from ..make_bg_cube_models import main as make_bg_cube_models_main
+from ...datasets import make_test_dataset
+
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+@pytest.mark.parametrize("extra_options,something_to_test", [
```

Comment:
I forgot to mention the most important points for such high-level analysis tests: we should introduce a marker for "slow" or "analysis" tests that don't get run on travis-ci, but that we run before making a release or weekly or nightly on a Jenkins I have at MPIK.
These can run real analyses, and it's OK if they take many minutes and in total even hours ... as long as they finish within a ~ day that's no problem.